index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
Perhaps we could move the original docs here.

Reply:
That's all up to you